role-based-access-system/src/app.js

```diff
@@ -0,0 +1,9 @@
+import express from "express";
+import route from "./routes/index.route.js";
```

Comment:
Are you sure this code is working cause i can't find index.js in routes folder anywhere